### PR TITLE
UX Updated Styling for "Ugly Baby" 

### PR DESCRIPTION
**Pull request by kfinlaysonosi**
@brunoOSI 
Updated HTML and added a new style sheet - can you review this for me? 

Notes: 
- The links need to be checked (some are direct and some are not so I may have made a mistake here) 
- I hard coded "pre-integration" as the environment name in this mock up but in the future we should ask dev-ops to create a variable for the environment name so that they can update automatically when used in different environments.

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -1,13 +1,161 @@
...
-    <%= link_to "Release Notes", @release_notes_url, {class:'btn btn-success' } %><br /><br />
-    <%= link_to "Job Aids" , @job_aids_url, {class:'btn btn-success' } %><br /><br />
-  </div >
+<div id="global-header">
+  <div class="container"> 
+    <div class="row">
+      <div class="col-md-12">
+        <div class="padded">
+          <span><strong>CWS-CARES</strong></span>
+          <span class="badge-environment half-gap-left">Pre-Intergration</span>
```
Comment (brunoOSI) [inline app/views/dashboard/index.html.erb]: Can we have some generic text for this until we get to the point where we can put the actual environment name?

File: app/views/dashboard/index.html.erb [hunk 2]
```diff
@@ -1,13 +1,161 @@
...
+      </div>
+    </div>
+  </div> 
+</div> 
+
+<div id="page-header">
+  <div class="container"> 
+    <div class="row">
+      <div class="col-md-12">
+        <h1>Pre-Integration Services<h1>
```
Comment (brunoOSI) [inline app/views/dashboard/index.html.erb]: same as above.. maybe a generic "Services"

File: app/views/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,13 +1,161 @@
...
+  </div> 
+</div> 
+
+
+<div class="container" id="main-content"> 
+  <div class="row">
+    <div class="col-md-12">
+      <div style="margin: 45px 0px 15px 0px">
+        <h3>Services</h3>
+        <p> The following are links to Pre-Integration environment. Due to continuous product improvements, content, colors and sizes are subject to change. Any and all feedback can be sent to Pierre Kleins personal email. He is totally cool with this. </p>
```
Comment (brunoOSI) [inline app/views/dashboard/index.html.erb]: and again

File: app/assets/stylesheets/dashboard.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    line-height: 60px;
+    padding-left: 6%;
+    margin-right: 15px;
+    float: left;
+}
+
+@font-face {
+    font-family: 'FontAwesome';
+    src: url("http://netdna.bootstrapcdn.com/font-awesome/4.6.3/fonts/fontawesome-webfont.eot?v=4.6.3");
+    src: url("http://netdna.bootstrapcdn.com/font-awesome/4.6.3/fonts/fontawesome-webfont.eot?#iefix&v=4.6.3") format("embedded-opentype"), url("http://netdna.bootstrapcdn.com/font-awesome/4.6.3/fonts/fontawesome-webfont.woff2?v=4.6.3") format("woff2"), url("http://netdna.bootstrapcdn.com/font-awesome/4.6.3/fonts/fontawesome-webfont.woff?v=4.6.3") format("woff"), url("http://netdna.bootstrapcdn.com/font-awesome/4.6.3/fonts/fontawesome-webfont.ttf?v=4.6.3") format("truetype"), url("http://netdna.bootstrapcdn.com/font-awesome/4.6.3/fonts/fontawesome-webfont.svg?v=4.6.3#fontawesomeregular") format("svg");
```
Comment (brunoOSI) [inline app/assets/stylesheets/dashboard.css]: we had some trouble before for CDN use. I'd recommend using the font-awesome gem: https://github.com/bokmann/font-awesome-rails

File: Gemfile.lock [hunk 5]
```diff
@@ -107,6 +107,7 @@ GEM
     minitest (5.10.1)
     multi_json (1.12.1)
     multipart-post (2.0.0)
+    newrelic_rpm (4.7.0.339)
```
Comment (brunoOSI) [inline Gemfile.lock]: why is new relic being added? is this related?

File: app/assets/stylesheets/dashboard.css (hunk 4 above)
Comment (kfinlaysonosi) [reply app/assets/stylesheets/dashboard.css]: Awesome, good to know!

File: Gemfile.lock (hunk 5 above)
Comment (kfinlaysonosi) [reply Gemfile.lock]: I did not add new relic....but let me look into it.
